e2e-tests/utils.sh

```diff
@@ -175,20 +175,33 @@ function debug_and_show_logs {
 
     echo "Flink logs:"
     kubectl get pods -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | while read pod;do
-        containers=(`kubectl get pods $pod -o jsonpath='{.spec.containers[*].name}'`)
-        i=0
-        for container in "${containers[@]}"; do
-          echo "Current logs for $pod:$container: "
-          kubectl logs $pod $container;
-          restart_count=$(kubectl get pod $pod -o jsonpath='{.status.containerStatuses['$i'].restartCount}')
-          if [[ ${restart_count} -gt 0 ]];then
-            echo "Previous logs for $pod: "
-            kubectl logs $pod $container --previous
-          fi
-        done
+        print_pod_container_logs "$pod" "{.spec.initContainers[*].name}" "{.status.initContainerStatuses[*].restartCount}"
+        print_pod_container_logs "$pod" "{.spec.containers[*].name}" "{.status.containerStatuses[*].restartCount}"
```

Comment:
I would add `echo "Printing init container logs"` and `echo "Printing main container logs"` for better understanding.